playwright/tina-playwright/tests/action/create-blog.spec.ts

```diff
@@ -1,13 +1,12 @@
 import { test, expect } from "@playwright/test";
-import deleteBlogPost from "../utils/deleteBlogPost";
+import deleteBlogPost from "../../utils/deleteBlogPost";
 
 test.describe("Create Blog Post", () => {
   test.beforeEach(async ({ page }) => {
     await page.goto(
       "http://localhost:3000/admin/index.html#/collections/new/post/~/",
```

Comment:
Will using hardcoded urls be an issue in any environment?

Reply:
There surely is a better way of doing this, but what will be your advice?

Reply:
Either, 

A. Add a constant that has the URL.
B. Not an issue - we can add a TODO.